constraint/pkg/client/drivers/local/driver_unit_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"reflect"
```

Comment:
Let's get the basic pattern down before writing tests. Changes in architecture will likely cause these tests to be thrown out.

Also, if we come up with a simpler model, the tests should be simpler to write.

If the concern is Test Driven Development... usually that requires knowledge of what the interface looks like, which is what we're trying to define.